quizapp/frontend-blog/src/redux/actions.js

```diff
@@ -0,0 +1,138 @@
+// Action Creators
+
+const setUser = (payload) => ({ type: "SET_USER", payload})
+const setBlogs = (payload) => ({ type: "SET_BLOGS", payload})
+
+export const logUserOut = () => ({type: "LOG_OUT"})
+
+// Methods
+
+export const getBlogs = () => dispatch => {
+    fetch(`http://localhost:8000/blog/api/blog/`, {
```

Comment:
please study about using the environment variables by using the .env file in react. and keep the domain base path in .env file. so that you dont need to update the base path while deploying the code on production (live server)